.github/workflows/build-package-test.yml

```diff
@@ -73,7 +75,7 @@ jobs:
           POSTGRES_VERSION: ${{ matrix.POSTGRES_VERSION }}
 
       - name: Clone tools repo for test
-        run: git clone -b v0.8.15 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b one_branch_additions --depth=1  https://github.com/citusdata/tools.git tools
```

Comment:
TODO: Branch name should be replaced with the new tools tag after tools PR merge

Reply:
Fixed